src/topp/FileConverter.cpp

```diff
@@ -143,6 +144,142 @@ String extractCachedMetaFilename(const String& in)
   return in_meta;
 }
 
+namespace OpenMS {
+enum IMType
```

Comment:
@timosachsenberg any idea where this should go? Where would be a good place to put this? E.g. `KERNEL/SpectrumHelper.h`?